README.md

```diff
@@ -1 +1,51 @@
-# Recipe-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Item catalog
+
+> The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
+
+<img src="./recipe_erd_2_members.png" alt="C=UML class diagram for catalog of things" />
```

Comment:
That's a great project! I like it. You both did a great job here!

Great documentation 🏆
No N+1 problem 🏆
No Linter errors 🏆

However great project comes with great responsibilities 😜. I'm just kidding. Or, I think I wanted to say, In big cities, there might be small problems as well. Ok, let's begin...